e2e/utils/zetacore.go

```diff
@@ -113,7 +113,7 @@ func WaitCctxsMinedByInboundHash(
 		allFound := true
 		for j, cctx := range res.CrossChainTxs {
 			cctx := cctx
-			if !IsTerminalStatus(cctx.CctxStatus.Status) {
+			if !cctx.CctxStatus.Status.IsTerminalStatus() {
```

Comment:
Let's  remove redundant suffixes
```suggestion
			if !cctx.CctxStatus.Status.IsTerminal() {
```